owncloudData/src/main/java/com/owncloud/android/data/migrations/Migration_31.kt

```diff
@@ -0,0 +1,29 @@
+/**
+ *   ownCloud Android client application
+ *
+ *   @author Abel García de Prada
+ *   Copyright (C) 2020 ownCloud GmbH.
+ *
+ *   This program is free software: you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License version 2,
+ *   as published by the Free Software Foundation.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+package com.owncloud.android.data.migrations
+
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
+
+val MIGRATION_30_31 = object : Migration(30, 31) {
+    override fun migrate(database: SupportSQLiteDatabase) {
+        //Nothing to migrate at the moment
```

Comment:
You could use // TODO and see it from Android studio bottom menu, to the right of `Debug` option

Reply:
It's not a TODO since there is nothing to migrate yet.